src/main.js

```diff
@@ -32,4 +44,16 @@ function renderInput(input, container) {
   container.appendChild(ulElement);
 }
 
-module.exports.render = renderInput;
+function renderUrl(url, container) {
+  fetch(url)
+    .then(resp => resp.json())
+    .then(function(data) {
+      const input = data;
+      renderInput(input, container);
+    })
+    .catch(function(error) {
+      console.log(error);
+    });
+}
+module.exports.renderUrl = renderUrl;
```

Comment:
We also need to test `renderUrl` in `main.spec.js`. Basically all we need to do is make sure that `renderUrl` will call `renderInput` with the JSON response received from `fetch`.

## Helpful links

- https://jestjs.io/docs/en/mock-function-api
- https://yarnpkg.com/en/package/jest-fetch-mock (you might need to add this with `yarn add --dev` to mock `fetch` also)